Dalamud/Interface/Internal/Windows/PluginInstallerWindow.cs

```diff
@@ -802,7 +845,7 @@ private bool DrawPluginListLoading()
             return ready;
         }
 
-        private bool DrawPluginCollapsingHeader(string label, PluginManifest manifest, bool trouble, bool updateAvailable, bool isNew, Action drawContextMenuAction, int index)
+        private bool DrawPluginCollapsingHeader(string label, LocalPlugin? plugin, PluginManifest manifest, bool isThirdParty, bool trouble, bool updateAvailable, bool isNew, Action drawContextMenuAction, int index)
```

Comment:
We need to pass around a nullable plugin so that we can detect devPlugins. For available plugins, we just pass in null here.